projects/1.21/assets/modular-routers/modularrouters/lang/zh_cn.json

```
@@ -0,0 +1,373 @@
+  "item.modularrouters.regex_filter": "正则过滤器",
+  "item.modularrouters.regulator_augment": "调节拓展",
+  "item.modularrouters.security_upgrade": "安全升级",
+  "item.modularrouters.sender_module_1": "发送模块 Mk1",
+  "item.modularrouters.sender_module_2": "发送模块 Mk2",
+  "item.modularrouters.sender_module_3": "发送模块 Mk3",
+  "item.modularrouters.speed_upgrade": "速度升级",
+  "item.modularrouters.stack_augment": "堆叠拓展",
+  "item.modularrouters.stack_upgrade": "堆叠升级",
+  "item.modularrouters.sync_upgrade": "同步升级",
+  "item.modularrouters.tag_filter": "标签过滤器",
+  "item.modularrouters.vacuum_module": "真空模块",
+  "item.modularrouters.void_module": "销毁模块",
+  "item.modularrouters.xp_vacuum_augment": "经验吸收拓展",
+  "key.modularrouters.category": "模块化路由器",
+  "key.modularrouters.configure": "配置已安装的模块（GUI）",
+  "key.modularrouters.moduleInfo": "额外的提示框信息（GUI）",
+  "modularrouters.chatText.misc.badCamoBlock": "无效的伪装方块！",
+  "modularrouters.chatText.misc.inventoryMerged": "已将%d个物品合并到%s中",
+  "modularrouters.chatText.misc.targetAdded": "目标已添加（%d/%d）：",
+  "modularrouters.chatText.misc.targetCleared": "目标已清空",
+  "modularrouters.chatText.misc.targetRemoved": "目标已移除（%d/%d）：",
+  "modularrouters.chatText.misc.targetSet": "目标已设定：",
+  "modularrouters.chatText.misc.tooManyTargets": "目标过多（最大：%d）",
+  "modularrouters.chatText.security.accessDenied": "§c--[拒绝访问]--",
+  "modularrouters.chatText.security.added": "已添加玩家[%s]",
+  "modularrouters.chatText.security.already_added": "玩家[%s]已存在于该升级中！",
+  "modularrouters.chatText.security.error": "无法添加玩家[%s]（内部错误）",
+  "modularrouters.chatText.security.full": "安全升级物品已满！",
+  "modularrouters.chatText.security.not_present": "玩家[%s]不在该升级中！",
+  "modularrouters.chatText.security.removed": "已移除玩家[%s]",
+  "modularrouters.chatText.subtitle.error": "错误",
+  "modularrouters.chatText.subtitle.success": "成功",
+  "modularrouters.chatText.subtitle.thud": "*砰的一声*",
+  "modularrouters.chatText.targetValidation.not_inventory": "✘ 无物品栏",
+  "modularrouters.chatText.targetValidation.not_loaded": "✘ 未加载",
+  "modularrouters.chatText.targetValidation.ok": "✔ 目标已标记",
+  "modularrouters.chatText.targetValidation.out_of_range": "✘ 超出范围",
+  "modularrouters.chatText.targetValidation.bad_dimension": "✘ 维度被列入黑名单",
+  "modularrouters._comment": "JEI text",
+  "modularrouters.configuration.Misc": "常规",
+  "modularrouters.configuration.Misc.tooltip": "常规客户端设置",
+  "modularrouters.configuration.Sound": "声音",
+  "modularrouters.configuration.Sound.tooltip": "声音设置",
+  "modularrouters.configuration.Module": "模块",
+  "modularrouters.configuration.Module.tooltip": "路由器模块设置",
+  "modularrouters.configuration.Router": "路由器",
+  "modularrouters.configuration.Router.tooltip": "路由器设置",
+  "modularrouters.configuration.Energy Costs": "模块能量消耗",
+  "modularrouters.configuration.Energy Costs.tooltip": "路由器消耗的FE",
+  "modularrouters.gui.config.alwaysShowSettings": "总是在模块提示文本中显示设置信息？",
+  "modularrouters.gui.config.baseTickRate": "路由器的基础运行间隔（无速度升级）（tick）",
```

| **审查级别** |
|:-:|
| ℹ️ 信息 / 不阻止合并  |

**原文: Base Router tick rate (no Speed Upgrades)**

```suggestion
  "modularrouters.gui.config.baseTickRate": "路由器的基础运行刻间隔（无速度升级）",
```